portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,27 +32,38 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private final List<String> comments = new ArrayList<>();
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("application/json;");
-    response.getWriter().println(convertToJsonUsingGson(comments));
-  }
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
This block for reading from datastore & populating the comments array can be moved to a separate ReadFromDatastore method;